helm fixes for latest/stable repos (#923)
rancher/docs: content/rancher/v2.x/en/installation/server-tags/_index.md

## High Availability Installs

When installing, upgrading, or rolling back Rancher Server in a [high availability configuration]({{< baseurl >}}/rancher/v2.x/en/installation/ha), you can choose what Helm repository from which to install your Helm chart.

### Helm Repositories

In high availability Rancher configurations, Rancher Server is distributed by a Helm chart. Therefore, as you prepare to install or upgrade a high availability Rancher configuration, you must configure a Helm repository that contains the charts for Rancher installations. You can install Rancher from two different repos:

Repository | Repo Configuration Command | Description
-----------|-----|-------------
`latest`   | `helm repo add rancher-latest https://releases.rancher.com/server-charts/latest` | Adds a repository of Helm charts for the latest versions of Rancher. We recommend using this repo for testing out new Rancher builds.
`stable`   | `helm repo add rancher-stable https://releases.rancher.com/server-charts/stable` | Adds a repository of Helm charts for older, stable versions of Rancher. We recommend using this repo for production environments.
<br/>

> Instructions on when to make choose these repos are available in [High Availability Install]({{< baseurl >}}/rancher/v2.x/en/installation/ha).

Reviewer: ```suggestion
Instructions on when to choose these repos are available in [High Availability Install]({{< baseurl >}}/rancher/v2.x/en/installation/ha).
```

#### Switching Helm Repositories

After installing Rancher, if you want to change which Helm repository to install Rancher from, you will need to follow these steps.

1. List the current Helm repos.

    ```
    helm repo list

    NAME          	      URL                                              
    stable        	      https://kubernetes-charts.storage.googleapis.com
    rancher-<CHART_REPO>	https://releases.rancher.com/server-charts/<CHART_REPO>
    ```

> 2. Remove the existing Rancher repo. This could either be `rancher-stable` or `rancher-latest` depending on what you had initially added.

Reviewer: ```suggestion
2. Remove the existing Rancher repo, which will either be `rancher-stable` or `rancher-latest` depending on your initial configuration.
```
